checkov/terraform/graph_builder/foreach_handler.py

```diff
@@ -243,3 +246,49 @@ def _create_new_foreach_resources(self, block_index_to_statement: FOR_EACH_BLOCK
             else:
                 self._create_new_resources_foreach(statement, self.local_graph.vertices[block_idx])
         self._delete_main_resource(block_index_to_statement)
+
+    def _handle_for_loop_in_foreach(self, block_idx: int, sub_graph: l_graph.TerraformLocalGraph) -> None:
+        block = sub_graph.vertices[block_idx]
+        foreach_statement = evaluate_terraform(block.attributes.get(FOREACH_STRING))
+        if not foreach_statement:
+            return
+
+        foreach_statement = self.extract_from_list(foreach_statement)
+        if isinstance(foreach_statement, str) and FOR_LOOP in foreach_statement:
+            if foreach_statement.startswith('${') and foreach_statement.endswith('}'):
```

Comment:
it's a bit inconsistent that you have consts defined for some strings and for some not (like `LEFT_BRACKET`).
can you extract them as well?